$NetBSD: patch-aa,v 1.4 2005/01/03 15:33:55 wennmach Exp $

Enable all routines (single & double precision, single and double
complex, auxilary routines in the "all" target. Enable libtool.

--- SRC/Makefile.orig	1999-11-04 20:26:09.000000000 +0100
+++ SRC/Makefile	2004-12-23 09:22:31.000000000 +0100
@@ -1,4 +1,4 @@
-include ../make.inc
+#include ../make.inc
 
 #######################################################################
 #  This is the makefile to create a library for LAPACK.
@@ -277,27 +277,11 @@
    zunmtr.o zupgtr.o \
    zupmtr.o izmax1.o dzsum1.o
 
-all: single complex double complex16
-
-single: $(SLASRC) $(ALLAUX) $(SCLAUX) 
-	$(ARCH) $(ARCHFLAGS) ../$(LAPACKLIB) $(SLASRC) $(ALLAUX) \
-	$(SCLAUX)
-	$(RANLIB) ../$(LAPACKLIB)
-
-complex: $(CLASRC) $(ALLAUX) $(SCLAUX)
-	$(ARCH) $(ARCHFLAGS) ../$(LAPACKLIB) $(CLASRC) $(ALLAUX) \
-	$(SCLAUX)
-	$(RANLIB) ../$(LAPACKLIB)
-
-double: $(DLASRC) $(ALLAUX) $(DZLAUX)
-	$(ARCH) $(ARCHFLAGS) ../$(LAPACKLIB) $(DLASRC) $(ALLAUX) \
-	$(DZLAUX)
-	$(RANLIB) ../$(LAPACKLIB)
-
-complex16: $(ZLASRC) $(ALLAUX) $(DZLAUX)
-	$(ARCH) $(ARCHFLAGS) ../$(LAPACKLIB) $(ZLASRC) $(ALLAUX) \
-	$(DZLAUX)
-	$(RANLIB) ../$(LAPACKLIB)
+all: $(ALLAUX) $(SCLAUX) $(DZLAUX) $(SLASRC) $(CLASRC) $(DLASRC) $(ZLASRC)
+	${LIBTOOL} --mode=link --tag=CC ${FC} -o ../liblapack.la            \
+	$(ALLAUX:.o=.lo) $(SCLAUX:.o=.lo) $(DZLAUX:.o=.lo) $(SLASRC:.o=.lo) \
+	$(DLASRC:.o=.lo) $(CLASRC:.o=.lo) $(ZLASRC:.o=.lo)                  \
+	-rpath ${PREFIX}/lib -version-info 3:0
 
 $(ALLAUX): $(FRC)
 $(SCLAUX): $(FRC)
@@ -313,10 +297,6 @@
 clean:
 	rm -f *.o
 
-slamch.o: slamch.f
-	$(FORTRAN) $(NOOPT) -c $<
-dlamch.o: dlamch.f
-	$(FORTRAN) $(NOOPT) -c $<
 
 .f.o: 
-	$(FORTRAN) $(OPTS) -c $<
+	${LIBTOOL} --mode=compile --tag=CC ${FC} ${FFLAGS} -c $<
